Support/injection.js

```diff
@@ -30,9 +30,12 @@ let observer = new IntersectionObserver(function(entries) {
 
 // register scroll view to handle heading on top of the page
 window.onscroll = function() {
-	if (document.documentElement.scrollTop <= 0) {
-		window.webkit.messageHandlers.headingVisible.postMessage({id: headings[0].id})
-	}
+    if (document.documentElement.scrollTop <= 0) {
```

Comment:
This was throwing some errors on certain pages, so I thought won't hurt to fix it.